finches.txt

```diff
@@ -0,0 +1,28 @@
+Observations of different attributes between finches from Santa Cruz, CA, San Cristobal, Mexico, and Floreana, Equador
```

Comment:
Floreana, San Cristobal, and Santa Cruz are all islands of the [Galapagos](https://en.wikipedia.org/wiki/Gal%C3%A1pagos_Islands#Main_islands)